src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -17,6 +17,19 @@ var assign = require('Object.assign');
 var emptyFunction = require('emptyFunction');
 var warning = require('warning');
 
+var didWarnForAddedNewProperty = false;
+var isSupportedProxy = typeof Proxy === 'function';
+
+var shouldBeReleasedProperties = [
```

Comment:
I think all these are fine, just noting this to myself (or others) because it's a potential change in the behavior of this class, but I don't think it has any impact on the overall program behavior.